test/lists.jl

```diff
@@ -0,0 +1,27 @@
+@testset "Polymake.StdList" begin
```

Comment:
there is some strange indentation here, could you make sure that 4 spaces are used instead of tabs?